gen/src/write.rs

```diff
@@ -236,13 +236,21 @@ fn write_struct<'a>(out: &mut OutFile<'a>, strct: &'a Struct, methods: &[&Extern
     writeln!(out, "#ifndef {}", guard);
     writeln!(out, "#define {}", guard);
     for line in strct.doc.to_string().lines() {
-        writeln!(out, "//{}", line);
+        if cfg!(feature = "doxygen") {
+            writeln!(out, "  ///{}", line);
```

Comment:
According to https://www.doxygen.nl/manual/docblocks.html, `///` is only considered Doxygen if there are at least 2 of them on consecutive lines. That's not how Rust `///` works normally so it would make sense for cxx-build to ensure it's converting Rust docs to Doxygen docs.

Reply:
This PR is not a complete implementation for generating the Doxygen docs. But a small enhancement that is enough for fullfilling the minimal doxygen docs generate requirement.

Two more lines are simple, just input two more lines in rust side.

Reply:
I understand that it could be simple for someone to work around, but I am not interested in accepting this PR as currently implemented. Taking Rust docs and turning them into either doxygen docs or silently into something that _looks_ like doxygen docs, but isn't, is going to come across as clearly broken behavior to anybody that uses this.

Reply:
Noted, let's find a better way to improve it later~